aiida/manage/external/rmq/client.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+"""Client for RabbitMQ Management HTTP API."""
+from __future__ import annotations
+
+import typing as t
+from urllib.parse import quote
+
+import requests
+
+from aiida.common.exceptions import AiidaException
+
+__all__ = ('RabbitmqManagementClient', 'ManagementApiConnectionError')
+
+
+class ManagementApiConnectionError(AiidaException):
```

Comment:
also this file looks like it could go to kiwipy

Reply:
Indeed, this is quite a generic API to interface with the management HTTP API. Could abstract this to a standalone library at some point or integrate into `kiwipy`.